pkg/resource/template/template_funcs.go

```diff
@@ -42,6 +45,20 @@ func addFuncs(out, in map[string]interface{}) {
 	}
 }
 
+// hashToIPv4 hashes a string for ipv6 only systems.
```

Comment:
Comment slightly out of date after the change.  Suggest `// hashToIPv4 hashes the given string and formats the resulting 4 bytes as an IPv4 address.`

Reply:
ack